src/System.Net.Security/System.Net.Security.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.26923.0
+# Visual Studio 14
```

Comment:
I'm geussing we don't want 14 and VS 15 will try to change it..

Reply:
I ran the target in a developer command prompt for VS 2017 so I wonder why this happened.

Reply:
It is hard coded in buildtools\src\Microsoft.DotNet.Build.Tasks\UpdateVSConfigurations.cs

Reply:
Hmm I see.  I can fix that later.

Reply:
yeah we can move that forward in the task.